nodejs/awsx/ec2/vpc.ts

```diff
@@ -702,10 +711,15 @@ export interface VpcArgs {
     subnets?: VpcSubnetArgs[];
 
     /**
-     * The maximum number of availability zones to use in the current region.  Defaults to `2` if
-     * unspecified.  Use `"all"` to use all the availability zones in the current region.
+     * The names of the availability zones to use in the current region. Defaults to `2` if
+     * unspecified. Use `"all"` to use all the availability zones in the current region.
+     */
+    requestedAvailabilityZones?: number | "all" | [string, ...string[]];
```

Comment:
The test is currently failing b/c it's passing in an `Output<string[]>` in when this doesn't accept that. I don't see an easy way to fix this except to change the test to not use `RandomShuffle` and use a different deterministic ordering instead, e.g. alphabetical.

Reply:
@leezen 
I see a few ways: 

- Maybe accept `Output<string[]>` 
- Hardcode region as I did initially, this removes need for `RandomShuffle`
- Use await to get a `string[]` - I need help here to understand how to add that in the unit tests.

Reply:
Added in https://github.com/pulumi/pulumi-awsx/pull/638/commits/c90df31b3497d709a044934377bb1c1d36401fbd